lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -155,52 +161,82 @@ export default function NarrativeItinerariesHeader({
             // because it falls under the "Plan your trip" <h1> header.
             <InvisibleA11yLabel as="h2">{itinerariesFound}</InvisibleA11yLabel>
           )}
-          <ItinerariesHeaderContainer showHeaderText={showHeaderText}>
+          <ItinerariesHeaderContainer
+            showHeaderText={showHeaderText}
+            style={{
+              alignItems: 'flex-end',
+              display: 'flex',
+              flexDirection: 'column',
+              width: '100%'
+            }}
+          >
             {popupTarget && (
               <button onClick={() => setPopupContent(popupTarget)}>
                 <PopupTriggerText compact popupTarget={popupTarget} />
               </button>
             )}
-            <button
-              className="clear-button-formatting"
-              onClick={onSortDirChange}
-              title={intl.formatMessage({
-                id: 'components.NarrativeItinerariesHeader.changeSortDir'
-              })}
-            >
-              <StyledIconWrapper
-                className={`${customBatchUiBackground && 'base-color-bg'}`}
+            {onClickExpansionButton && (
+              <button
```

Comment:
This button needs to have a `title` and an `aria-label` for non-sighted users. You can repurpose the i18n strings for "expand map" and "show results"